arbeitszeit/use_cases/__init__.py

```diff
@@ -98,8 +98,9 @@
     QueryCompanies,
     QueryCompaniesRequest,
 )
+from .query_company_purchases import QueryCompanyPurchases
+from .query_member_purchases import PurchaseQueryResponse, QueryMemberPurchases
```

Comment:
I would welcome it if with future PRs the imports from this module would be removed instead of fixed since we don't decided some time ago that maintaining this module is unnecessary work.